src/main/java/org/sagebionetworks/web/client/SynapseJavascriptClient.java

```diff
@@ -1534,16 +1533,20 @@ public void initSession(String token) {
 	}
 
 	/**
-	 * call to ask the server to set the session cookie
+	 * call to ask the server to set the access token cookie
 	 * 
 	 * @param token
 	 * @param callback
 	 */
 	public void initSession(String token, AsyncCallback<Void> callback) {
-		Session s = new Session();
-		s.setSessionToken(token);
-		String url = jsniUtils.getSessionCookieUrl();
-		doPost(url, s, OBJECT_TYPE.None, false, callback);
+		AccessTokenWrapper t = new AccessTokenWrapper();
```

Comment:
Created my own object to wrap the token.  It still has the same structure, but I don't rely on a backend json object that may go away.  Maintaining the structure (key is still "sessionToken") makes my life easier since other portals hosted on .synapse.org domains use the InitSessionServlet in www.synapse.org to access the current session token.  Eases migration from session token to access token for other portals.